contracts/tokens/NFTokenEnumerable.sol

```diff
@@ -56,6 +56,7 @@ contract NFTokenEnumerable is
   {
     super._mint(_to, _tokenId);
     tokens.push(_tokenId);
+    idToIndex[_tokenId] = tokens.length.sub(1);
```

Comment:
`push` function returns new array length, so I tend to do:
```
uint length = tokens.push(_tokenId);
idToIndex[_tokenId] = length - 1;
```
Don't know if that's more gas efficient tho.